modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -47,34 +41,38 @@ anypoint.platform.gatekeeper=flexible
 
 === Security Level Definitions
 
+You can apply one of the listed security levels to Gatekeepr:
+
 ==== False
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process. This value is available only for backward compatibility in Mule 3.x. Use ‘disabled’ instead. `disabled` is not available in all 3.x releases. `false` is not available in Mule 4.x.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘disabled’ instead for all other scenarios. Note that the `disabled` value is not available in all Mule 3.x releases and the `false` value is not available in Mule 4.x releases.
 
 ==== Disabled
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied.
 
 ==== True
 
-When this value is set, the default security level is Strict. This value is available only for backward compatibility in Mule 3.x. Use ‘strict’ or ‘flexible’ instead. `true` is unavailable in Mule 4.x
+When you set this value, the default security level is  set to `Strict`. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘strict’ or ‘flexible’ instead for all other scenarios. The value `true` is unavailable in Mule 4.x releases.
 
 ==== Strict
 
-When this value is established, Gatekeeper security is set on its maximum security level.
+When you set this value, Gatekeeper security is set on its maximum security level.
 
-When the Mule Runtime is (re)started, the API is blocked and returns a 503 HTTP status code, until policies for each tracked resource are correctly downloaded and applied. If connectivity with API Platform is lost when Gatekeeper is engaged, the tracked resource will be blocked at least, until connection is re-established.
+When the Mule instance is (re)started, the API is blocked and a "503 - Service Unavailable" HTTP status code is returned until policies for each tracked resource are correctly applied. If the connectivity to Anypoint Platform is lost when Gatekeeper is engaged, the tracked resource is blocked until the connection is reestablished.
```

Comment:
```suggestion
When the Mule instance is restarted, the API is blocked with `503 - Service Unavailable` until policies for each tracked resource are correctly applied. If connectivity to Anypoint Platform is lost when Gatekeeper is engaged, the tracked resource is blocked until the connection is reestablished.
```
I think we can shorten the text around 503 now that it has been established....

Reply:
Done! Thanks!